### ISSUE TITLE
Boardwalk for android 9

### ISSUE THREAD

**Issue by Just1-N00B** [zhuowei/Boardwalk#362]
Hi I was just wondering if there was a version of boardwalk for android 9.

**Comment by Un1q32** [comment]
@Just1-N00B No, but I found this that accomplishes the same goal.  It's in a different language but still usable if you don't understand it.  Worked on Android 7.1.2 Google pixel sailfish, which boardwalk doesn't currently support, give it a try
https://drive.google.com/file/d/1JOC1VS63yOQWv1tk6C42lg6vQH3q3aQK/view

**Comment by kran27** [comment]
That's actually just a translation of an already existing launcher, which is in English to begin with.  Can't remember where I found the link to it, but I have it on my phone

**Comment by Un1q32** [comment]
@kran27 what is the name of the launcher?  Can you upload the apk to google drive and share the link?  I would very much like to use the english version.

**Comment by ghost** [comment]
I found the SRC code for Qlauncher https://github.com/KamenkoTV/Qlauncher-mcpc-android

**Comment by 12brendon34** [comment]
its a RIP off of kdudetrans or whatever just google net.minecraft.kdt apk and download it

**Comment by 12brendon34** [comment]
this is soo dead i wold of had loved for a remake or even pay

**Comment by khanhduytran0** [comment]
You didn't see any updates on YouTube, package has changed.
